### PR TITLE
Added a suggestion for the purchase price to the documentation

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**

### CODE REVIEW

**Review by luisaterrabrito — DISMISSED** [review]

File: src/06-products/products.apib [hunk 1]
```diff
@@ -24,3 +24,5 @@ Get a list of products.
                         + name: `cookies` (string)
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
+                + purchase_price (Money)
```
Comment (dcsg) [inline src/06-products/products.apib]: I believe you forgot to build the documentation it should also add it to the `https://github.com/teamleadercrm/api/blob/master/apiary.apib`

**Review by dcsg — CHANGES_REQUESTED** [review]

File: src/06-products/products.apib (hunk 1 above)
Comment (EOSullivanBerlin) [reply src/06-products/products.apib]: Its built now.

**Review by dcsg — DISMISSED** [review]

**Review by luisaterrabrito — DISMISSED** [review]

**Review by Yarn-e — DISMISSED** [review]

**Review by jdrieghe — DISMISSED** [review]

**Review by dcsg — DISMISSED** [review]

File: src/06-products/products.apib [hunk 2]
```diff
@@ -24,3 +24,5 @@ Get a list of products.
                         + name: `cookies` (string)
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
+                + purchase_price (Money, optional)
```
Comment (dcsg) [inline src/06-products/products.apib:27]: I believe this can be `null` but not `optional`

Comment (EOSullivanBerlin) [reply src/06-products/products.apib:27]: Okay what should we do? I am a bit lost.

**Pull request closed without merging**